devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy

```diff
@@ -85,6 +85,7 @@ metadata {
 
 		// Muller Licht Tint
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 1000", outClusters: "0019", manufacturer: "MLI", model: "ZBT-ColorTemperature", deviceJoinName: "Müller Licht Tint White Bulb"
+        fingerprint profileId: "0104", deviceId: "010C", manufacturer: "AduroSmart Eria", model: "AD-ColorTemperature3001", deviceJoinName: "Eria Color Temperature Light"
```

Comment:
@adurosmart  can you add a section for your brand similar to the other fingerprints in this DTH?

Reply:
Thank you review.
What is DTH? I don't understand the meaning of DTH?

Reply:
DTH is short for Device Type Handler, referring to this file

Reply:
also tabs instead of spaces to match the rest of the file